frontend/app/src/i18n/en.json

```diff
@@ -829,7 +829,11 @@
             "loginApprover": "Sorry we were unable to sign into your current OpenChat authentication provider",
             "loginInitiator": "Sorry we were unable to sign into your chosen provider",
             "pollingError": "There was an unexpected error waiting for email signin",
-            "principalMismatch": "It seems like you did not re-authenticate with the same provider that you are currently signed in with"
+            "principalMismatch": "It seems like you did not re-authenticate with the same provider that you are currently signed in with",
+            "unlinkActivePrincipal": "It seems like you tried to unlink a provider you're currently logged in with, which is not allowed.",
```

Comment:
I think "You cannot unlink the provider you are currently logged in with." is better.